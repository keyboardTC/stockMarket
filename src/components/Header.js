import { StyleSheet, Text, View } from 'react-native'
import React from 'react'

const Header = props => {
    return (
        <View style={[styles.mainContainer, props.style]}>
            <Text style={[styles.titleText, props.textStyle]}>{props.text}</Text>
        </View>
    )
}

export default Header

const styles = StyleSheet.create({
    mainContainer: {
        backgroundColor: 'pink',
        justifyContent: 'center',
        alignItems: 'center'
    },
    titleText: {
        fontWeight: 'bold',
        fontSize: 28,
        marginVertical: 10,
        marginHorizontal: 20
    }
})