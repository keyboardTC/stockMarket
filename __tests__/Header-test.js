import 'react-native';
import React from 'react';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

import Header from '../src/components/Header';

// Header Component Test
it('renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
});