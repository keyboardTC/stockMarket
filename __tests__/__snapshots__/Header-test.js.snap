// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "pink",
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 28,
          "fontWeight": "bold",
          "marginHorizontal": 20,
          "marginVertical": 10,
        },
        undefined,
      ]
    }
  />
</View>
`;
